docs/src/main/paradox/jms.md

```diff
@@ -172,6 +172,29 @@ Scala
 Java
 : @@snip ($alpakka$/jms/src/test/java/akka/stream/alpakka/jms/javadsl/JmsConnectorsTest.java) { #create-messages-with-headers }
 
+
+### Sending messages as a Flow
+
+The producer can also act as a flow, in order to publish messages in the middle of stream processing.
+For example, you can ensure that a message is persisted to the queue before subsequent processing.
```

Comment:
"producer" is kafka terminology and I don't think it means anything in general for people using JMS. What about something along the lines of "Sending messages to a queue can also be done in a flow where the elements are passed downstream after they have been persisted allowing for further processing"